subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -568,15 +564,17 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 				&params->bssid[4], &params->bssid[5]);
 			break;
 		case 'h':
-			shell_help(sh);
+			return -ENOEXEC;
```

Comment:
Why this change?

Reply:
Without this change, wifi ap  enable -h will print the help then configure an AP with the default SSID an no security